stablehlo/tests/vhlo/stablehlo_legalize_to_vhlo.0_10_0.mlir

```diff
@@ -5,6 +5,10 @@
 // RUN: diff %t.0 %t.1
 // RUN: stablehlo-translate --serialize --target=0.10.0 --strip-debuginfo %s > %t.2
 // RUN: diff %s.bc %t.2
+// RUN: stablehlo-opt --stablehlo-legalize-to-vhlo -emit-bytecode -debug-only=vhlo-bytecode %s 2>&1 | FileCheck --check-prefix=CHECK-WARN %s
+// RUN: stablehlo-opt --stablehlo-legalize-to-vhlo -emit-bytecode %s | stablehlo-opt -debug-only=vhlo-bytecode 2>&1 | FileCheck --check-prefix=CHECK-WARN %s
```

Comment:
Why not:

`// RUN: stablehlo-opt -debug-only=vhlo-bytecode %s 2>&1 | FileCheck --check-prefix=CHECK-WARN %s`

Reply:
Sure, we can do that too. I grabbed these from `stablehlo_legalize_to_vhlo.mlir`.

Reply:
Actually IMO that doesn't belong here since this is a test for VHLO legalization. But we should test this somewhere, I'll send a PR to test this in `ops_stablehlo`.

Reply:
Follow-up: go/shlopr/2080